geocity/apps/core/static/js/submissions_submit.js

```diff
@@ -0,0 +1,5 @@
+let formNumber = document.querySelector("[data-geo-time-role='formNumber']");
+
+if (formNumber.textContent == "1") {
+  formNumber.style.display = "none";
+}
```

Comment:
Already done in submissions_geo_time.js line 14
https://github.com/yverdon/geocity/commit/68ba81e14524680c74c65e1dac1d05a4f7e8907a

Reply:
Yes but we're not in the same context `this.formNumber.textContent`.

And more of that I didn't want to load all the js file just for that.